### PR TITLE
add customize kibana documentation

### PR DESCRIPTION
**Pull request by sebartyr**
How to customize Kibana configuration

### CODE REVIEW

File: deploy/addon/elastic/kibana.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+## Overview
+
+[Kibana](https://www.elastic.co/fr/kibana/) is the web interface available on the Clever Cloud's platform to manage your Elastic Stack.
+
+By default, you can enable Kibana when you create an Elastic Stack add-on.
+
+## Enable Kibana with your Elastic Stack add-on
+
+Kibana can be enabled at the add-on creation. Choose "Create an add-on" > "Elastic Stack". Select your plan, applications which will use the Elasctic Stack and name the add-on. On the options for the Elastic Stack page, enable Kibana. Then, confirm options and your add-on is ready with Kibana enabled.
```
Comment (judu) [inline deploy/addon/elastic/kibana.md:26]:
```suggestion
Kibana can be enabled at the add-on creation. Choose "Create an add-on" > "Elastic Stack". Select your plan, applications which will use the Elastic Stack and name the add-on. On the "options" step, enable Kibana. Then, confirm the options and your add-on will start with a Kibana instance.
```

File: deploy/addon/elastic/kibana.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+### Customize Kibana configuration file
```
Comment (judu) [inline deploy/addon/elastic/kibana.md:28]:
```suggestion
### Customize the Kibana configuration file
```

File: deploy/addon/elastic/kibana.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+The configuration is set with this deployment hook :
+`CC_PRE_RUN_HOOK` = `curl https://api.clever-cloud.com/v2/providers/es-addon/kibana-setup/<your elastic version> | sh`
+
+To modify this default configuration ([Configuration file for Kibana 8.3.3](https://api.clever-cloud.com/v2/providers/es-addon/kibana-setup/8.3.3)), you need to host your own config file (we strongly recommend [Cellar](/deploy/addon/cellar))
```
Comment (judu) [inline deploy/addon/elastic/kibana.md:33]:
```suggestion
To modify this default configuration ([Configuration file for Kibana 8.3.3](https://api.clever-cloud.com/v2/providers/es-addon/kibana-setup/8.3.3)), you need to host your own config file (we strongly recommend [Cellar](/deploy/addon/cellar)).
```

File: deploy/addon/elastic/kibana.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+Check other available configuration file on [Github](https://github.com/sebartyr/custom-kibana-config)
```
Comment (judu) [inline deploy/addon/elastic/kibana.md:35]: Push this to a Clever Cloud repository.

File: deploy/addon/elastic/kibana.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+### Disable SSO authentication
+
+To disable SSO authentication and use instead elastic users, you need to modify the configuration file of Kibana and `CC_PRE_RUN_HOOK` in environment variables.
```
Comment (judu) [inline deploy/addon/elastic/kibana.md:39]:
```suggestion
To disable SSO authentication and use elastic users instead, you need to modify Kibana's configuration file and `CC_PRE_RUN_HOOK` in environment variables.
```

File: deploy/addon/elastic/kibana.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+For example for Kibana 8.3.3: 
+`CC_PRE_RUN_HOOK` = `curl https://raw.githubusercontent.com/sebartyr/custom-kibana-config/master/no-sso-8.3.3 | sh`
```
Comment (judu) [inline deploy/addon/elastic/kibana.md:42]: Link to the Clever Cloud repository.

File: deploy/addon/elastic/kibana.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+Remember, you need to ask Clever Cloud Support team to enable superuser permissions to your user. Then, you could add some additional users with Kibana.
```
Comment (judu) [inline deploy/addon/elastic/kibana.md:44]:
```suggestion
Remember, you need to ask Clever Cloud Support team to grant superuser permissions to your user. After that, you will be able to add additional users via Kibana.
```

File: deploy/addon/elastic/kibana.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+### Add custom domain name
+
+You need to disable SSO authentication first. Then, you will be able to add a custom domain name in "Domain name" tab of your Kibana app.
+
+## Deploy Kibana at localhost
```
Comment (judu) [inline deploy/addon/elastic/kibana.md:50]:
```suggestion
## Deploy Kibana on localhost
```

File: deploy/addon/elastic/kibana.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+Follow these steps :
+- Download/unzip Kibana in version which match with your Elastic Stack version
+- Edit kibana.yml :
+    - line 43 : `elasticsearch.host: <elastic-addon-host>:443` 
+    - line 49 : `elasticsearch.username: kibana`
+    - line 50 : `elasticsearch.password: <password>`
+- Launch Kibana (`<path_to_kibana>/bin/kibana`)
+- Go on Kibana through the dedicated [local address](http://localhost:5601)
```
Comment (judu) [inline deploy/addon/elastic/kibana.md:59]:
```suggestion
- Go to Kibana through the dedicated [local address](http://localhost:5601)
```

**Review by judu — CHANGES_REQUESTED** [review]

File: deploy/addon/elastic/kibana.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+To modify this default configuration ([Configuration file for Kibana 8.3.3](https://api.clever-cloud.com/v2/providers/es-addon/kibana-setup/8.3.3)), you need to host your own config file (we strongly recommend [Cellar](/deploy/addon/cellar)).
+
+Check other available configuration file on [Github](https://github.com/sebartyr/custom-kibana-config)
+
+### Disable SSO authentication
+
+To disable SSO authentication and use elastic users instead, you need to modify Kibana's configuration file and `CC_PRE_RUN_HOOK` in environment variables.
+
+For example for Kibana 8.3.3: 
+`CC_PRE_RUN_HOOK` = `curl https://raw.githubusercontent.com/sebartyr/custom-kibana-config/master/no-sso-8.3.3 | sh`
```
Comment (sebartyr) [inline deploy/addon/elastic/kibana.md:42]:
```suggestion
`CC_PRE_RUN_HOOK` = `curl https://raw.githubusercontent.com/CleverCloud/custom-kibana-config/master/no-sso-8.3.3 | sh`
```

File: deploy/addon/elastic/kibana.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+
+Remember, you need to ask Clever Cloud Support team to grant superuser permissions to your user. After that, you will be able to add additional users via Kibana.
+
+### Add custom domain name
+
+You need to disable SSO authentication first. Then, you will be able to add a custom domain name in "Domain name" tab of your Kibana app.
+
+## Deploy Kibana on localhost
+
+Follow these steps :
```
Comment (sebartyr) [inline deploy/addon/elastic/kibana.md:52]:
```suggestion
## Deploy Kibana on localhost

The Kibana version should match with the ElasticSearch version.

Follow these steps :
```

File: deploy/addon/elastic/kibana.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+To modify this default configuration ([Configuration file for Kibana 8.3.3](https://api.clever-cloud.com/v2/providers/es-addon/kibana-setup/8.3.3)), you need to host your own config file (we strongly recommend [Cellar](/deploy/addon/cellar)).
+
+Check other available configuration file on [Github](https://github.com/CleverCloud/custom-kibana-config)
+
+### Disable SSO authentication
+
+To disable SSO authentication and use elastic users instead, you need to modify Kibana's configuration file and `CC_PRE_RUN_HOOK` in environment variables.
+
+For example for Kibana 8.3.3: 
+`CC_PRE_RUN_HOOK` = `curl https://raw.githubusercontent.com/CleverCloud/custom-kibana-config/master/no-sso-8.3.3 | sh`
```
Comment (sebartyr) [inline deploy/addon/elastic/kibana.md:42]:
```suggestion
`CC_PRE_RUN_HOOK` = `curl https://raw.githubusercontent.com/CleverCloud/custom-kibana-config/master/8.3.3/no-sso-8.3.3 | sh`
```

**Pull request merged**